app/public/js/common/playerService.js

```diff
@@ -271,19 +271,30 @@ app.factory('playerService', function($rootScope, $log, $timeout, notificationFa
      * Responsible to add scrubbing
      * song progress bar
      */
-    $(player.elPlayerProgress).parent().off().on('click', function(e) {
-        var percent = ( e.offsetX / $(this).width() );
-        var duration = player.elPlayer.duration;
-        var seek = percent * duration;
+    var scrub = $(player.elPlayerProgress).parent().off();
 
-        if ( player.elPlayer.networkState === 0 || player.elPlayer.networkState === 3 ) {
-            notificationFactory.error("Something went wrong. I can't play this track :(");
-        }
+    scrub.on('mousedown', function (e) {
```

Comment:
The click event happens on the mouseup partion, so it wouldn't do trigger the event correctly.